osu.Game/Screens/OnlinePlay/DrawableRoomPlaylistItem.cs

```diff
@@ -448,31 +430,34 @@ protected override Drawable CreateContent()
             };
         }
 
-        private IEnumerable<Drawable> createButtons() => new[]
+        private IEnumerable<Drawable> createButtons()
         {
-            showResultsButton = new GrayButton(FontAwesome.Solid.ChartPie)
-            {
-                Size = new Vector2(30, 30),
-                Action = () => RequestResults?.Invoke(Item),
-                Alpha = AllowShowingResults ? 1 : 0,
-                TooltipText = "View results"
-            },
-            Item.Beatmap.Value == null ? Empty() : new PlaylistDownloadButton(Item),
-            editButton = new PlaylistEditButton
-            {
-                Size = new Vector2(30, 30),
-                Alpha = AllowEditing ? 1 : 0,
-                Action = () => RequestEdit?.Invoke(Item),
-                TooltipText = "Edit"
-            },
-            removeButton = new PlaylistRemoveButton
+            return new[]
```

Comment:
Just a preferred styling change here?

Reply:
It originally had more implementation, will revert to original.